recipe/meta.yaml

```diff
@@ -12,13 +12,14 @@ source:
 build:
   number: 0
   script:
-    - pip install --no-deps .
+    - python setup.py install
     - python -m ipykernel install --prefix $PREFIX  # [not win]
     - python -m ipykernel install --prefix %PREFIX%  # [win]
 
 requirements:
   build:
     - python
+    - setuptools
```

Comment:
When we get Python from `conda-forge` `setuptools` is not available and things that import `pkg_resources` are broken. I am still puzzled why `conda-forge` is going in the opposite direction and removing `setuptools` and `pip` by default instead of bundling them.

See https://travis-ci.org/ocefpaf/ipykernel-feedstock/jobs/151533092#L848